ethminer/main.cpp

```diff
@@ -344,6 +344,9 @@ class MinerCLI
         app.add_option("--cuda-streams,--cu-streams", m_CUSettings.streams, "", true)
             ->check(CLI::Range(1, 99));
 
+        app.add_option("--cuda-usleep,--cu-usleep", m_CUSettings.usleep, "", true)
+                    ->check(CLI::Range(-1, 1000000));
```

Comment:
It's probably worth improving the documentation on this, just to make it more clear what this actually does.

```suggestion
        // adds an optional usleep between batches. Can be used to slow down mining.
        app.add_option("--cuda-usleep,--cu-usleep", m_CUSettings.usleep, "Sleep time in μs between batches (default: no sleep)", true)
            ->check(CLI::Range(-1, 1000000));
```